Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -238,6 +241,18 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
         }
       }
 
+    self.contentInset = self.viewWillAppearProperty.signal
```

Comment:
Is this because the old cards had some padding the new ones don't?

Reply:
Yep, the new cards have very little padding so they need a bit more on the top and bottom of the table view.